iroh-net/Cargo.toml

```diff
@@ -79,6 +79,8 @@ tracing-subscriber = { version = "0.3", features = ["env-filter"], optional = tr
 # metrics
 iroh-metrics = { version = "0.7.0", path = "../iroh-metrics", default-features = false }
 bounded_join_set = "0.1.0"
+ttl_cache = "0.5.1"
+serde_bytes = "0.11.12"
```

Comment:
we can also write the weird visitors ourselves if you don't want the dependency. But it's small and from @dtolnay, so I'd rather not.